src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/RightCurly/context/Test.java

```diff
@@ -0,0 +1,32 @@
+package TreeWalker.blocks.RightCurly;
+
+public class Test {
+    public void test() {
+        int x = 0;
+        while (true) {
+            try {
+                if (x > 0)
+                {
+                    break;
+                }
+                else if (x < 0)
+                {
+                    ;
+                }  // violation context
+                else
+                {
+                    break;
+                }
+
+            }
+            catch (Exception e)
+            {
+                break;
```

Comment:
add changes to this line as well. There should be a violation because of not aligned `catch`

Reply:
done